src/Models/Constants/Interfaces.ts

```diff
@@ -141,10 +142,26 @@ export interface IConsumeCompositeCardProps extends ICardBaseProps {
 }
 
 export interface IAuthService {
-    login: () => void;
-    getToken: (
-        tokenFor?: 'azureManagement' | 'adx' | 'storage' | 'powerBI'
-    ) => Promise<string>;
+    /** the unique object id of the logged in user */
+    userObjectId?: string;
+    setActiveAccount: (account: any) => void;
+    login: (continuation?: (params: any) => void) => void;
+    logout: () => void;
+    getToken: (tokenFor?: AuthTokenTypes) => Promise<string>;
+}
+
+export interface IAuthParams {
+    /** base login url for the auth service authority */
+    login: string;
+    clientId: string;
+    tenantId?: string;
+}
+
+export interface IMSALLoginContinuationParams {
+    name: string;
+    userName: string;
+    tenants: Array<any>;
```

Comment:
can we get any more specific than `any`? Can it be Record<string, string> or something? Do we at least know it's an object?

Reply:
based on [getTenants ](https://learn.microsoft.com/en-us/rest/api/resources/tenants/list?tabs=HTTP)response I added the interface which has the keys that I am interested in.